hedyorg/hedy
# [UI] Make newsletter sign up only for teachers & add new question to sign-up form only for teachers

**Description**
In this PR, we make some changes to the sign-up form:
- The "subscribe to the newsletter" option is only available for teachers signing up and NOT students signing up.
- The "how did you hear about Hedy?" question and its options is only available for teachers signing up and NOT students signing up.

**Fixes**
This PR fixes #3769 

**How to test**

- Try signing up as a student to verify that they can't see the newsletter option and new section.
- Try signing up as a teacher a verify that they can see the newsletter option and new section.
- Run tests_e2e.py (specifically test_invalid_signups() and test_signup()) to test back-end functionality.
